### ISSUE TITLE
Medulloblastoma subtype propagation question

### ISSUE THREAD

**Issue by jharenza** [AlexsLemonade/OpenPBTA-analysis#758]
#### What data file(s) does this issue pertain to?
pbta-histologies.tsv
#757 

#### What release are you using?
Preparation for #732 

#### Put a link to the relevant section of the [OpenPBTA-manuscript](https://github.com/AlexsLemonade/OpenPBTA-manuscript/) here.
NA

#### Put your question or report your issue here.
MB BS_5BMNK8SY DNA has two associated RNA samples: BS_QEYPFYHD (classifiers did not agree) and BS_Z7PKVY9J (classifiers agreed == Group4). In #444, we propagated RNA subtypes to DNA labels per #379, but this sample would create an issue with doing so. How should we proceed? These biospecimens are from the same event ID (7316-14) and are both from the Initial CNS tumor, so I _**think**_ it could be safe to add logic as the following:
```
IF there are two RNA specimens from the same event (`sample_id`) with the same `tumor_descriptor`, and 
IF one of the two RNA specimens has a consensus match from the MB classifiers, and 
IF the RNA specimen with mismatched classifications includes a match to the subtype of the consensus of the other RNA specimen, 
THEN propagate the subtype to the second RNA specimen.
```
😬 ?

cc @jaclyn-taroni @komalsrathi @baileyckelly

**Comment by jharenza** [comment]
> @jharenza can you clarify what you mean by
> 
> ```
> the RNA specimen with mismatched classifications includes a match to the subtype of the consensus of the other RNA specimen
> ```

Sure - example: 

**Situation 1**
| Specimen   | MB-classifier | MM2S classifier |
|------------|---------------|-----------------|
| Specimen A | subtype X     | subtype X       |
| Specimen B | subtype X     | subtype Y       |

vs. 

**Situation 2**
| Specimen   | MB-classifier | MM2S classifier |
|------------|---------------|-----------------|
| Specimen A | subtype X     | subtype X       |
| Specimen B | subtype Y     | subtype Z       |

Talking about situation 1.

**Comment by jaclyn-taroni** [comment]
If the end result is that the DNA specimen in situation 1 gets subtype X and in situation 2 the DNA specimen would be unclassified, what you are proposing seems reasonable.

**Comment by jharenza** [comment]
> If the end result is that the DNA specimen in situation 1 gets subtype X and in situation 2 the DNA specimen would be unclassified, what you are proposing seems reasonable.

Yes, that is my thought. Thanks - I will ask @komalsrathi to update the subtyping module for this one sample.

**Comment by jaclyn-taroni** [comment]
Closed via #760
